eventstream.go

```diff
@@ -35,8 +35,9 @@ const (
 	eventStreamStdout = "stdout"
 	eventStreamKafka  = "kafka"
 
-	actorTypeUser   = "USER"
-	actorTypeClient = "CLIENT"
+	ActorTypeADMIN  = "ADMIN"
```

Comment:
I'm not sure why you introduced the ADMIN and made the category make no sense. Previously the client means the request is invoked by client credentials and the user is invoked by user. Admin belong to user.

Reply:
> I'm not sure why you introduced the ADMIN and made the category make no sense. Previously the client means the request is invoked by client credentials and the user is invoked by user. Admin belong to user.

This is for account history, we need to know the actor is the user or an admin